### ISSUE TITLE
Update unit tests to phantomjs 2.0

### ISSUE THREAD

**Issue by sathomas** [sathomas/STEM#40]
Something in the tool chain doesn't seem to be working with the latest phantomjs 2.0 release. Probably ought to fix that to avoid confusion.

**Comment by jcarroll2007** [comment]
Is it possible to give a little more information related to this ticket? Where can I see the errors?

Thanks!

**Comment by jcarroll2007** [comment]
Is the fact that `blanket.js` shows 100% test coverage one of the problems? It seems to show 100% coverage on the test overview page, but when you drill down the coverage changes to the correct percentage. (I believe.)

**Comment by sathomas** [comment]
Might be a problem with my local environment, but when I run `grunt test` it never completes. Browser-based testing via `grunt serve:test` works as expected. We do have 100% coverage for all included code, so that value is correct.

**Comment by jcarroll2007** [comment]
![image](https://cloud.githubusercontent.com/assets/8485105/7302355/d220000e-e9b6-11e4-8810-c4d066c2cfd2.png)

Looks like it's working ok on my end.

**Comment by sathomas** [comment]
ok. I'll mark this as invalid but leave it open until I figure out what's going on on my machine

**Comment by jcarroll2007** [comment]
Oh ok, I was confused about the coverage because when I select one of the groups of tests (e.g. Tag Models), blanket.js changes the coverage of each of the files. For example: The Partners Models coverage changes to ~7% coverage. Is this expected behavior?

**Comment by sathomas** [comment]
Sounds like you're just testing the Partners Model, which is 7% of the total code base that's under test.

**Comment by jcarroll2007** [comment]
Hm.. I'm not quite sure that that's the reason why. Here is what the blanket results look like when I select a specific test file. 
![image](https://cloud.githubusercontent.com/assets/8485105/7302447/7e5a3f9c-e9b7-11e4-804e-294726b9d4a2.png)

**Comment by sathomas** [comment]
Depends on what you did to create that screen shot, but it might be plausible if you're just running a subset of the tests. Unit tests for some modules necessarily execute code in other modules. In fact, some models, such as Poi and Search, have no code at all; they're just standard Backbone models. In those cases, the 4 statements that are covered are just the 4 lines of code that define the model. By definition, they'll be considered covered with any subset of tests.

The bottom line is that if all the tests are run, the coverage should be 100%.

**Comment by jcarroll2007** [comment]
Oh, that makes sense. Thanks for the clarification.

**Comment by sathomas** [comment]
Suggest moving this issue to https://github.com/Georgia-STEM-Incubator/STEM if it's still a problem
